docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,1335 @@
+| $$ H_{gasLimit} > 5000$$  |  calculate_base_fee_per_gas -> [check_gas_limit](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L1132)     | |      |        |            |      |
+| $$H_{timeStamp} > P(H)_{H_{timeStamp'}} $$  | validate_header-> [ensure](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L323)      | |      |        |            |      |
+| $$ H_{numberOfAncestors} = P(H)_{H_{numberOfAncestors'}} + 1 $$  |  validate_header-> [ensure](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L324)            | |      |        |            |      |
+| $$ length(H_{extraData}) \leq 32_{bytes} $$  |    validate_header-> [ensure](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L325)                 | |      |        |            |      |
+| $$ H_{baseFeePerGas} = F(H) $$  |       | |      |        |            |      |
+| $$ H_{parentHash} = KEC(RLP( P(H)_H ))  $$  |        validate_header-> [ensure](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L332)                           | |      |        |            |      |
+| $$ H_{ommersHash} = KEC(RLP(())) $$  |       validate_header-> [ensure](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L329)                     | |      |        |            |      |
+| $$ H_{difficulty} = 0 $$  |      validate_header-> [ensure](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L327)               | |      |        |            |      |
+| $$H_{nonce} = 0x0000000000000000 \tag{57l}$$  |       validate_header-> [ensure](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L328)                     | |      |        |            |      |
+| $$  H_{prevRandao} = PREVRANDAO() \text{this is stale , beacon chain provides this now} $$  |       | |      |        |            |      |
+| $$ H_{withdrawlsHash} \neq nil $$  |       | |      |        |            |      |
+| $$ H_{blobGasUsed} \neq nil $$  |       | |      |        |            |      |
+| $$ H_{blobGasUsed} \leq  MaxBlobGasPerBlock_{=786432}  $$  |       | |      |        |            |      |
+| $$ H_{blobGasUsed} \% GasPerBlob_{=2^{17}} = 0 $$  |       | |      |        |            |      |
+| $$ H_{excessBlobGas} = CalcExcessBlobGas(P(H)_H) $$  |  [ensure ](https://github.com/ethereum/execution-specs/blob/9fc7925c80ff2f3949e1cc340a4a0d36fcd4161c/src/ethereum/cancun/fork.py#L187)     | |      |        |            |      |
+
+# Block Execution Process
```

Comment:
don't use multiple h1s please

Reply:
Shall I break the H1's into separate files ?

Reply:
Not necessary but the doc should have a structure, e.g.:
- h1, only one main title (Execution Layers Specification)
- h2s, different parts of the client (e.g. Block validation)
- h3s, subtopics of h2s (e.g. validating individual tx)